jobs/index.html

```diff
@@ -18,8 +22,12 @@
   var jobs = null
 
   var jobContainer = document.getElementById('jobs')
-  var jobItem = function (contents) {
-    return '<li>' + contents + '</li>'
+  var jobItem = function (element) {
+    date = new Date(element.created_at)
+    var dateOptions = {
+      year: "numeric", month: "short", day: "numeric"
+    };
+    return '<a class="list-group-item"><h4><span class="job-title">' + element.title + '</span><span class="badge pull-right">' + date.toLocaleDateString("en-US", dateOptions) + '</span></h4></a>'
```

Comment:
this needs to be cleaned up to use string interpolation better or just something ... input appreciated (cc @theetrain )

Reply:
I don't think we're allowed to wrap an `h4` within an `a`. We'll clean this up.

Reply:
I was shamelessly stealing the design from here http://demo.jobskee.com/